Client/Application/LaunchScreen.xib

```diff
@@ -18,7 +21,7 @@
                     </constraints>
                 </imageView>
             </subviews>
-            <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="calibratedWhite"/>
+            <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
```

Comment:
Thanks for the PR! Can you unstage the other changes if this is the only needed change? Thanks

Reply:
Should be good now!